drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -90,22 +90,22 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     )
 
     weight = Float(
-        display_name=_("Weight"),
-        help=_("The maximum score the learner can receive for the problem"),
+        display_name=_("Maximum score"),
+        help=_("The maximum score the learner can receive for the problem."),
         scope=Scope.settings,
         default=1,
     )
 
     item_background_color = String(
         display_name=_("Item background color"),
-        help=_("The background color of draggable items in the problem."),
+        help=_("The background color of draggable items in the problem (e.g. 'blue' or '#0000ff')."),
```

Comment:
Should we use "example" instead of e.g.? I am not sure how well e.g. translates.